src/modules/jcms_rest/tests/src/Functional/RamlSchemaValidationTest.php

```diff
@@ -12,30 +12,47 @@
 use GuzzleHttp\Psr7\Response;
 use GuzzleHttp\Client;
 use JsonSchema\Validator;
+use RuntimeException;
 
 /**
  * @group jcms_rest
  * @todo Make this work with KernelTestBase instead of UnitTestCase.
  */
-class RamlSchemaValidationTest extends UnitTestCase {
+class RamlSchemaValidationTest extends FixtureBasedTestCase {
 
   /**
    * @var \GuzzleHttp\Client
    */
   protected $client;
 
-  protected $projectRoot;
-
   /**
    * @var MessageValidator
    */
   protected $validator;
 
   protected static $contentGenerated = FALSE;
 
+  public static function setUpBeforeClass()
+  {
+    parent::setUpBeforeClass();
+    // Generate content once.
+    if (!self::$contentGenerated) {
+      self::$contentGenerated = TRUE;
+      $projectRoot = realpath(__DIR__ . '/../../../../../..');
+      $script = $projectRoot . '/scripts/generate_content.sh';
+      if (!file_exists($script)) {
+        throw new RuntimeException("File $script does not exist");
+      }
+      $logFile = '/tmp/generate_content.log';
+      exec("$script >$logFile 2>&1", $output, $exitCode);
+      if ($exitCode != 0) {
+        throw new RuntimeException("$script failed. Check log file $logFile");
```

Comment:
`symfony/process` is available, can simplify this.

Reply:
It is not in the explicit requirements in composer.json, so I would avoid introducing additional dependencies in place of 3-4 lines of standard library calls